### DISCUSSION TITLE
Windows 10

### DISCUSSION THREAD

**Discussion by luibgn** [bochs-emu/Bochs#399] [Q&A]
I installed windows 10 x64 in bochs, just for fun because it is too slow to be really usable.
But it runs!
I used the vbe video adapter.
The E1000 network card is working (not the NE).
But I have no sound. The ES1370 is not recognized.
Suggestions? Maybe we need a new sound system for modern OS.
A new challenge for Ruppert.

**Comment by Vort** [comment]
No sound in Windows 7 x64 as well.
These OSes require emulation of more modern hardware for full-fledged operation.
This is massive work however. _(Especially, emulation of relatively modern GPUs)_
